hyprbars/barDeco.cpp

```diff
@@ -281,6 +281,10 @@ void CHyprBar::renderBarTitle(const Vector2D& bufferSize, const float scale) {
     cairo_surface_destroy(CAIROSURFACE);
 }
 
+inline constexpr std::uint32_t fnv1a(const char* str, std::uint32_t hash = 2166136261UL) {
```

Comment:
excuse me?

Reply:
https://en.wikipedia.org/wiki/Fowler%E2%80%93Noll%E2%80%93Vo_hash_function

this is the most obscure thing i have ever seen??
maybe you can understand it better but wtf.

Reply:
You got me, I'm bad at c/c++ and  this is the only way I could make it work. But this is nice feature and it would be cool if you add it (in a better way of course)

Reply:
ok but idek what using a hash function would achieve in this scenario, like, why not simply 
`cpp
 static auto* const PBUTTONSFORM      = (Hyprlang::STRING const*)HyprlandAPI::getConfigValue(PHANDLE, "plugin:hyprbars:buttons_form")->getDataStaticPtr();
switch(tolower(pbuttonsform))
{
  case "square":
    // stuff
  break;
  //etc
}
 `
 ?

Reply:
Yeah, it is stupid, I got it. Thanks for providing correct way to do it, I will use it in the future. I close request, but it's still good option to have. Please add it